### ISSUE TITLE
Can not Login - Server cannot append header after HTTP headers have been sent.

### ISSUE THREAD

**Issue by siavashfdst** [jakubgarfield/Bonobo-Git-Server#338]
I´m having this error when I try to login. The error appears in the Bonobo.Git.Server.Errors.log only after trying to login with admin/admin for the first time:
The page reloads and nothing happens

    Native library pre-loader is trying to load native SQLite library       "C:\inetpub\wwwroot\gitserver\bin\x64\SQLite.Interop.dll"...
    w3wp.exe Error: 0 : Server Error occured and caught in Global.asax - System.Web.HttpException    (0x80004005): Server cannot append header after HTTP headers have been sent.
       at System.Web.HttpHeaderCollection.SetHeader(String name, String value, Boolean replace)
       at Microsoft.Owin.Host.SystemWeb.CallHeaders.AspNetResponseHeaders.Set(String key, String[]   values)

**Comment by AndreuGenestar** [comment]
I'm having the same problem, and the same error on Bonobo.Git.Errors.log 
When I try to login for the first time (admin/admin), it reloads the page and writes that error.
When I try to login with a wrong user(xxxx/xxxx) it works fine displaying "The username or password provided is incorrect".

Any idea?

**Comment by larshg** [comment]
Have you guys even looked at already opened issues :) ? ie. #329 #330 ? Anyways there should be a fix in PR #335 ;)

**Comment by AndreuGenestar** [comment]
Thanks!

**Comment by jakubgarfield** [comment]
Duplicate of #329 #330

**Comment by mmcer** [comment]
can u help to release the fixed version to the website, that we can download the binary file directly?

**Comment by Aimeast** [comment]
加群，有方案，200319579
